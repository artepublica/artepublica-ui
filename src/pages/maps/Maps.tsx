import { StyleSheet, View } from 'react-native';

import { Text } from '@components';

function Maps(): JSX.Element {
    return (
        <View style={styles.container}>
            <Text>Maps</Text>
        </View>
    );
}

const styles = StyleSheet.create({
    container: {
        flex: 1,
        alignItems: 'center',
        justifyContent: 'center',
    },
});
export default Maps;
