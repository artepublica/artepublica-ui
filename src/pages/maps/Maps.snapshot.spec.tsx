import { create } from 'react-test-renderer';

import Maps from './Maps';

describe('<Maps />', () => {
    it('default snapshot', () => {
        const json = create(<Maps />).toJSON();

        expect(json).toMatchSnapshot();
    });
});
