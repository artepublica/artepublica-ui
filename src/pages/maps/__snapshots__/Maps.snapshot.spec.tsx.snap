// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Maps /> default snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "white",
          "fontFamily": "fontFamily",
        },
        undefined,
      ]
    }
  >
    Maps
  </Text>
</View>
`;
